test_requirements.txt

```diff
@@ -1,5 +1,6 @@
 # Core project/common application testing requirements
 
+django-celery
```

Comment:
I don't think this is required. I'm running the test suite locally without it and tests are passing.

The [django-celery](https://pypi.python.org/pypi/django-celery) docs say it's deprecated, and the [celery docs](http://celery.readthedocs.org/en/latest/django/first-steps-with-django.html) say you only need it if you want to use the Django ORM as the results backend, but we're using Redis as the results backend.